common/cloudant_base.go

```diff
@@ -101,6 +103,13 @@ func NewBaseService(opts *core.ServiceOptions) (*BaseService, error) {
 		return &BaseService{}, err
 	}
 	client := core.DefaultHTTPClient()
+	if client.Jar == nil {
+		jar, err := cookiejar.New(&cookiejar.Options{PublicSuffixList: publicsuffix.List})
```

Comment:
```suggestion
		client.Jar, err := cookiejar.New(&cookiejar.Options{PublicSuffixList: publicsuffix.List})
```
not a biggie, but should safe a line

Reply:
Fixed in [f87eb79](https://github.com/IBM/cloudant-go-sdk/pull/398/commits/f87eb79a1ee5173f386fd8b201df860047335456)